cmd/explaintest/r/explain_generate_column_substitute.result

```diff
@@ -304,26 +304,36 @@ desc update t set a=1 where a+1 = 3;
 id	estRows	task	access object	operator info
 Update_4	N/A	root		N/A
 └─IndexLookUp_11	10.00	root		
-  ├─IndexRangeScan_9(Build)	10.00	cop[tikv]	table:t, index:expr_idx_c(_V$_expr_idx_c_0)	range:[3,3], keep order:false, stats:pseudo
+  ├─IndexRangeScan_9(Build)	10.00	cop[tikv]	table:t, index:expr_idx_c(`a` + 1)	range:[3,3], keep order:false, stats:pseudo
   └─TableRowIDScan_10(Probe)	10.00	cop[tikv]	table:t	keep order:false, stats:pseudo
 desc update t set a=2, b = 3 where b+a = 3;
 id	estRows	task	access object	operator info
 Update_4	N/A	root		N/A
 └─IndexLookUp_11	10.00	root		
-  ├─IndexRangeScan_9(Build)	10.00	cop[tikv]	table:t, index:expr_idx_e(_V$_expr_idx_e_0)	range:[3,3], keep order:false, stats:pseudo
+  ├─IndexRangeScan_9(Build)	10.00	cop[tikv]	table:t, index:expr_idx_e(`b` + `a`)	range:[3,3], keep order:false, stats:pseudo
   └─TableRowIDScan_10(Probe)	10.00	cop[tikv]	table:t	keep order:false, stats:pseudo
 desc delete from t where a+1 = 3;
 id	estRows	task	access object	operator info
 Delete_4	N/A	root		N/A
 └─IndexLookUp_11	10.00	root		
-  ├─IndexRangeScan_9(Build)	10.00	cop[tikv]	table:t, index:expr_idx_c(_V$_expr_idx_c_0)	range:[3,3], keep order:false, stats:pseudo
+  ├─IndexRangeScan_9(Build)	10.00	cop[tikv]	table:t, index:expr_idx_c(`a` + 1)	range:[3,3], keep order:false, stats:pseudo
   └─TableRowIDScan_10(Probe)	10.00	cop[tikv]	table:t	keep order:false, stats:pseudo
 desc delete from t where b+a = 0;
 id	estRows	task	access object	operator info
 Delete_4	N/A	root		N/A
 └─IndexLookUp_11	10.00	root		
-  ├─IndexRangeScan_9(Build)	10.00	cop[tikv]	table:t, index:expr_idx_e(_V$_expr_idx_e_0)	range:[0,0], keep order:false, stats:pseudo
+  ├─IndexRangeScan_9(Build)	10.00	cop[tikv]	table:t, index:expr_idx_e(`b` + `a`)	range:[0,0], keep order:false, stats:pseudo
```

Comment:
It's better to use the expression like `b+a` in the explain results. The normalized string takes too many unnecessary symbols, such as "`" and space.

Reply:
This string comes from tableInfo.GeneratedExprString field, I think it can be used directly.
Using them after format maybe is a little complicated? Do you have any good suggestions ?

Reply:
![image](https://user-images.githubusercontent.com/30121694/95968461-0afd4b80-0e40-11eb-954a-407b43175693.png)